homeassistant/components/device_tracker/huawei_router.py

```diff
@@ -78,7 +74,6 @@ def get_device_name(self, device):
                 return client.name
         return None
 
-    @Throttle(MIN_TIME_BETWEEN_SCANS)
```

Comment:
I've experienced some weird results when using too frequent updates (like 1 second or so), so I think this should stay to prevent that.